jwcrypto/jwa.py

```diff
@@ -1104,6 +1101,7 @@ def signing_alg(cls, name):
         try:
             return cls.instantiate_alg(name, use='sig')
         except KeyError:
+            # pylint: disable=raise-missing-from
```

Comment:
Python 3 does exception chaining. I recommend to add ``raise InvalidJWAAlgorithm(...) from None`` unless you want to show the inner exception to your users.